validation/custom_constraint.rst

```diff
@@ -250,3 +250,73 @@ not to the property:
                 $metadata->addConstraint(new ProtocolClass());
             }
         }
+
+How to Unit Test your Validator
+-------------------------------
+
+To create a unit test for you custom validator, your test case class should
+extend the ``ConstraintValidatorTestCase`` class and implement the ``createValidator()`` method::
+
+    protected function createValidator()
+    {
+        return new ContainsAlphanumericValidator();
```

Comment:
What about using the `ProtocolClassValidator` here that we implemented above?

Reply:
From what I see, the `ProtocolClassValidator` is more like 'sketchy' snippet, no real logic in there. Ergo I am not sure that we can produce some significant unit test example for it. Maybe we should improve the `ProtocolClassValidator` snippet? @xabbuh

Reply:
That would work for me too. As long as we use a consistent example throughout this document the experience when reading the article will be better IMO.

Reply:
@henry2778 it would be very great if you could use (and enhance) the `ProtocolClassValidator` to stay consistent across the document. Thanks